generated/nidcpower/session.py

```diff
@@ -2378,6 +2378,55 @@ def fetch_multiple(self, count, timeout=datetime.timedelta(seconds=1.0)):
         the *NI DC Power Supplies and SMUs Help* for more information about
         configuring this method.
 
+        Note: This method is not supported on all devices. Refer to `Supported Methods by Device <REPLACE_DRIVER_SPECIFIC_URL_2(nidcpowercref.chm, supportedfunctions)>`__ for more information about supported devices.
+
+        Tip:
+        This method requires repeated capabilities (usually channels). If called directly on the
+        nidcpower.Session object, then the method will use all repeated capabilities in the session.
+        You can specify a subset of repeated capabilities using the Python index notation on an
+        nidcpower.Session instance, and calling this method on the result.:
+
+            session.channels['0,1'].fetch_multiple(count, timeout=datetime.timedelta(seconds=1.0))
+
+        Args:
+            count (int): Specifies the number of measurements to fetch.
+
+            timeout (datetime.timedelta): Specifies the maximum time allowed for this method to complete, in seconds. If the method does not complete within this time interval, NI-DCPower returns an error.
+
+                Note: When setting the timeout interval, ensure you take into account any triggers so that the timeout interval is long enough for your application.
+
+
+        Returns:
+            measurements (list of float): Returns an array of named tuples.
+
+        '''
+        import collections
+        Measurements = collections.namedtuple('Measurements', ['voltage', 'current', 'in_compliance'])
+
+        voltage_measurements, current_measurements, in_compliance = self._fetch_multiple(count, timeout)
+
+        measurements = []
+        for i in range(count):
+            measurements.append(Measurements(voltage=voltage_measurements[i], current=current_measurements[i], in_compliance=in_compliance[i]))
+
+        return measurements
+
+    def _fetch_multiple(self, count, timeout=datetime.timedelta(seconds=1.0)):
```

Comment:
Couldn't count have a default as well? It's 1 in the LV API. 
(I would have added this comment to fetch_multiple above, but apparently github doesn't allow me to comment on unchanged code)

Reply:
I was thinking along the same lines, but would make the default be the backlog.
I didn't make a comment because of our policy: https://github.com/ni/nimi-python/wiki/API-design-guidelines#default-values-for-methods

Reply:
Are you testing with a simulated device? On some implementations the backlog will always return that one measurement is available.

Reply:
Yes, we do our system tests with simulated devices. This particular test is simulating a PXIe-4162.

I can simulate a different one for this test if it will help. What would be an alternative?